Specs/DateTime/German/DateTimeModel.json

```diff
@@ -208,7 +208,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "NotSupported": "python",
+    "NotSupported": "",
```

Comment:
In such cases, the NotSupported attribute should be removed, instead of left empty in all specs.